config/links.coffee

```diff
@@ -33,3 +33,4 @@ module.exports =
     "supervisor-api": 'https://github.com/balena-io/balena-supervisor/edit/master/docs/API.md'
     "update-locking": "https://github.com/balena-io/balena-supervisor/edit/master/docs/update-locking.md"
     "diagnostics": "https://github.com/balena-io/device-diagnostics/edit/master/diagnostics.md"
+    "google": "https://github.com/balenalabs/google-iot/edit/master/README.md"
```

Comment:
I wonder if `google` might be a bit too generic, and using `google-iot` might be better? :man_shrugging:

Reply:
Yeh, this needs to be the same as the filename (which already exists as `google.md`) and I was trying to avoid a redirect. That being said `cli` is already causing me headaches due to being too generic so this may be worth implementing now.